pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -41,12 +44,21 @@ func New(profile *performancev2.PerformanceProfile) (*machineconfigv1.KubeletCon
 			"cpu":    defaultSystemReservedCPU,
 			"memory": defaultSystemReservedMemory,
 		},
+		FeatureGates: map[string]bool{
```

Comment:
It will not work, the only way to enable feature gates under the OpenShift is via FeatureGate CR